.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@
 
 - [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
 - [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
-- [ ] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides]. (Note that not like the flutter/flutter repo, we do use dart formatter in the flutter/plugins repo.)
+- [ ] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides]. (Note that not like the flutter/flutter repo, we do use dart formatter in the flutter/plugins repo. Run `dart run ./script/tool/lib/src/main.dart format` under plugins/ to auto format all source code.)
```

Comment:
> not like the -> unlike the

Reply:
done

Reply:
This will only work if they have done a `pub get` there. I think we should put instructions in CONTRIBUTING.md, and then link directly to that part of it from here.